documents/openapi/openapi.yaml

```diff
@@ -2510,9 +2570,64 @@ components:
       example:
         "full_name": "John Smith"
         "claim_id": "123456"
+    InboundNumberResponseData:
+      type: object
+      properties:
+        active:
+          type: boolean
+        auth_parameter:
+          type: string
+        id:
+          type: string
+          format: uuid
+        number:
+          type: string
+        provider:
+          type: string
+        self_managed:
+          type: boolean
+        service:
+          type: object
+          properties:
+            id:
+              type: string
+              format: uuid
+            name:
+              type: string
+          nullable: true
+        url_endpoint:
+          type: string
+    InboundNumberRequestData:
+      type: object
+      properties:
+        active:
+          type: boolean
+        auth_parameter:
+          type: string
+        number:
+          type: string
+        provider:
+          type: string
+        self_managed:
+          type: boolean
+        service_id:
+          type: string
+          format: uuid
+          nullable: true
+        url_endpoint:
+          type: string
     InboundNumberCreateResponse:
       type: object
-      properties: {} # TODO (issue 965)
+      properties:
+        data:
+          $ref: "#components/schemas/InboundNumberResponseData"
```

Comment:
This line is causing three validation errors. 
Please update as shown, with a `/ `after `#` and before `components`
`$ref: "#/components/schemas/InboundNumberResponseData"`

Reply:
updated